packages/react/src/hooks/useAuth.ts

```diff
@@ -117,6 +124,7 @@ export const useAuth: UseAuth = () => {
       orgSlug: undefined,
       signOut,
       getToken,
+      authStateKey: undefined,
```

Comment:
Shouldn't we use a hash here as well? It is still an "authState".

Reply:
🔧 
It'd expect `authStateKey` to  be a `string` always so it needs to take a value here so it can be used to bust caches when going from loaded -> signed in or loaded -> signed out states

Reply:
@desiprisg Maybe we could just hash an empty string and have that as its value

Reply:
Why can't we do exactly as we do in the other cases?